docs/conf.py

```diff
@@ -13,21 +13,28 @@
 
 # pylint: disable=invalid-name
 
-import os
+import re
 import sys
+from datetime import datetime
+from importlib.metadata import metadata
```

Comment:
This was added to the Python std lib? How do you feel about only supporting `>=py3.8`. It is possible to support more versions of Python with this backport lib, https://pypi.org/project/importlib-metadata/. But personally, I say good riddance to older Python's lol. FWIW, 3.6 is EOL this year.

Reply:
What about 3.7?

Reply:
specifically, what about 3.7? 
> It is possible to support more versions of Python with this backport lib, https://pypi.org/project/importlib-metadata/. But personally, I say good riddance to older Python's lol.

Reply:
Just wondering because you mentioned only supporting 3.8 or greater but 3.6 is eol. So we skipped a number (3.7) and was wondering why.

Reply:
Cuz https://docs.python.org/3/whatsnew/3.8.html#new-modules . And not deal w/ including a backport to support an older version of Py. If it is that large of a concern, we can. I just dont see the need myself.